packages/turbo-ignore/tsup.config.ts

```diff
@@ -3,7 +3,7 @@ import { defineConfig, Options } from "tsup";
 export default defineConfig((options: Options) => ({
   entry: ["src/index.ts"],
   format: ["cjs"],
-  minify: true,
+  minify: false,
```

Comment:
why change this? wouldn't a minified build download faster from npm?

Reply:
Yes! Leftover from a bundling test I was doing. Thanks for catching 🙇🏼